src/Application/ConfigResolver.php

```diff
@@ -177,4 +178,37 @@ private static function excludeGlobalInsights(array $config): array
 
         return $config;
     }
+
+    /**
+     * @param array<string, array|string|int> $preset
+     * @param array<string, array|string> $config
+     *
+     * @return array<string, array|int|string>
+     */
+    private static function preparePreset(array $preset, array $config): array
+    {
+        $removedRulesByPreset = [];
+        $addedRulesByConfig = [];
+
+        if (isset($preset['remove'])) {
```

Comment:
I think this key is always present in $preset, maybe we should add more check ? 

For example : 

```php
if (isset($preset['remove']) && is_array($preset['remove']) && count($preset['remove']) > 0) {
```

Reply:
@Jibbarth No problem. Thanks a lot. And yes, this PR is exactly what you've mentioned.

Reply:
Yep, you are right :+1: